lib/nostrum/struct/voice_ws_state.ex

```diff
@@ -16,4 +18,65 @@ defmodule Nostrum.Struct.VoiceWSState do
     :heartbeat_interval,
     :heartbeat_ref
   ]
+
+  @typedoc "The guild id"
+  @type guild_id :: Nostrum.Struct.Guild.id()
+
+  @typedoc "The session id"
+  @type session :: String.t()
+
+  @typedoc "The session token"
+  @type token :: String.t()
+
+  @typedoc "PID of the `:gun` worker connected to the websocket"
+  @type conn :: pid
+
+  @typedoc "PID of the connection process"
+  @type conn_pid :: pid
+
+  @typedoc "Stream reference for `:gun`"
+  @type stream :: reference()
+
+  @typedoc "Gateway URL"
+  @type gateway :: String.t()
+
+  @typedoc "Where the session has been identified"
```

Comment:
I believe there's a typo here:
```suggestion
  @typedoc "Whether the session has been identified"
```